routes/curations.js

```diff
@@ -52,10 +52,51 @@ router.get(
 router.get(
   '/:type?/:provider?/:namespace?/:name?',
   asyncMiddleware(async (request, response) => {
-    const coordinates = utils.toEntityCoordinatesFromRequest(request)
-    return curationService.list(coordinates).then(result => response.status(200).send(result))
-  })
-)
+      const coordinates = utils.toEntityCoordinatesFromRequest(request)
+      curationService.list(coordinates).then(function(result){
+
+        // access the request url and method
```

Comment:
@moranthomas I am unclear on why we are adding this code as part of this PR. Is this existing code and wrong formatting causes a false positive?

Reply:
So yeah this is not strictly part of this PR - as I assumed since there was no data coming back from curations as we had to mock it I would leave this in as well. In hindsight I agree that we should keep this separate so I will remove this code and re-commit